common.mk

```diff
@@ -28,6 +28,7 @@ PYTHON3 ?= python3
 ROOT ?= $(shell pwd)/..
 
 UNAME_M				:= $(shell uname -m)
+TARGET_ARCH			?= arm
```

Comment:
Could this be just `$(ARCH)`? I know the Linux kernel uses the same name and may need a different value (the Linux `ARCH` may be `arm` or `arm64` when this `TARGET_ARCH` is `arm`) but I don't think it is an issue at all since `ARCH` is explicitly provided in `LINUX_*_FLAGS`.
What do others think?

Reply:
I am OK about this

Reply:
Changed in latest commits